gridstatus/miso.py

```diff
@@ -22,6 +24,7 @@ class MISO(ISOBase):
 
     name = "Midcontinent ISO"
     iso_id = "miso"
+    # TODO: what does this next line mean?
```

Comment:
These next two lines seem to contradict each other.

Reply:
i agree...something is off here...